task_master.go

```diff
@@ -267,6 +269,10 @@ type TaskMaster struct {
 	wg      sync.WaitGroup
 }
 
+func (tm *TaskMaster) WritePointsPrivileged(ctx tsdb.WriteContext, database, retentionPolicy string, consistencyLevel imodels.ConsistencyLevel, points []imodels.Point) error {
```

Comment:
Are we adding this to implement an interface? Can we adjust whatever requires that interface to not require unused methods?

Reply:
It is an interface in influxdb 1.9.6 where it is actually used.

Reply:
Does kapa allow `SELECT INTO` ? Do you get a good error message if you try to use that?

Reply:
It does allow it but it just the same as having run it on the target influx machine